mkosi.md

```diff
@@ -528,10 +528,24 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 
 `Verity=`, `--verity`
 
-: Add an "Verity" integrity partition to the image. If enabled, the
-  root partition is protected with `dm-verity` against off-line
-  modification, the verification data is placed in an additional GPT
-  partition. Implies `ReadOnly=yes`.
+: Add a "Verity" integrity partition to the image. Takes a boolean or
+  the special value `signed`, and defaults to disabled. If enabled,
+  the root partition (or `/usr/` partition, in case `UsrOnly=` is
+  enabled) is protected with `dm-verity` against offline modification,
+  the verification data is placed in an additional GPT
+  partition. Implies `ReadOnly=yes`. If this is enabled, the Verity
+  root hash is written to an output file with `.roothash` or
+  `.usrhash` suffix. If set to `signed`, Verity is also enabled, but
+  the resulting root hash is then also signed (in PKCS#7 format) with
+  the signature key configured with `SecureBootKey=`. Or in other
+  words: the SecureBoot key pair is then both used to sign the kernel,
+  if that's enabled, and the root/`/usr/` file system. This signature
```

Comment:
Bikeshedding and a formatting fix
```suggestion
  words: the SecureBoot key pair is then used to both sign the kernel,
  if that is enabled, and the `root`/`/usr/` file system. This signature
```

Reply:
Applied, except for "root" → it shouldn't be quoted, because it's a name, not a path.